README.md

```diff
@@ -346,6 +351,35 @@ command:
 xcodebuild build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -project "Mozilla VPN.xcodeproj"
 ```
 
+#### Building with CMake (Experimental)
+
+We also support building from sources for iOS using CMake. This is a work in progress and these
+instructions may be subject to change as we figure out what works and what doesn't.
+
+1. On iOS, we compile the app using
+[Xcode](https://developer.apple.com/xcode/) version 12 or higher and [Qt](https://www.qt.io/download)
+version 6.3.2.
+
+2. We use `qt-cmake` from the Qt installation to configure the Xcode project.
+```bash
+/Users/example/Qt/6.3.2/ios/bin/qt-cmake . -B build -GXcode
+```
```

Comment:
The GH suggestion syntax conflicts with the readme bash syntax, hence the dashes thrown in here - will need to remove.

```suggestion
2. Use `qt-cmake` (from the Qt installation) to build and configure the Xcode project:
```bash
/Users/example/Qt/6.3.2/ios/bin/qt-cmake . -B build-ios -GXcode
-```

This builds for an iOS device. Due to cmake limitations, we need to include a build variable for building for the simulator. To make it easier to develop, it's recommended to build to a different folder/project for simulator, so that switching between device and simulator testing does not require re-building the project:
```bash
/Users/example/Qt/6.3.2/ios/bin/qt-cmake . -B build-ios-sim -GXcode CMAKE_OSX_SYSROOT="iphonesimulator"
-```
```